bagit.xml

```diff
@@ -597,48 +627,41 @@ Internal-Sender-Description: Uncompressed greyscale TIFFs created
           <t>
             For reasons of efficiency, a bag &may; be sent with a list of files to be
             fetched and added to the payload before it can meaningfully be checked
-            for completeness.  An &optional; tag file named "fetch.txt"
+            for completeness.  An &optional; tag file called the fetch file
             contains such a list.
           </t>
 
           <t>
-            Every file listed in fetch.txt &must; be listed in every payload manifest.
+            The fetch file &must; be named "fetch.txt". Every file listed in
+            the fetch file &must; be listed in every
+            payload manifest. A fetch file &must-not; list any tag files.
```

Comment:
Why can't a Fetch file list tag files? This is especially confusing since the fetch path is relative to the base directory and thus if it is a payload file must start with `/data`

Reply:
It would need to explicitly exclude manifests so you know what you're getting. Perhaps that would say “&must-not; list any standard tag files but &may; list tag files which are not defined in this spec”?

Reply:
fetch.txt has always been described as listing payload files. Without a compelling reason, I'd be reluctant to change it now. (But would have no objection > 1.0.)

Reply:
I know users have used this in the past to fetch more than just payload files which is why I brought it up. Why are you reluctant to change it? It doesn't change the format of the file and adds functionality that users have used in the past.

Reply:
It would require a "starting in 1.0 ..." since it was prohibited in the past. @acdha thoughts?

Reply:
I'm a strong -1 against allowing it to contain tag manifests since that breaks the security model. Other than that, I don't feel very strongly but don't see much point in supporting the regular manifests. It would probably be useful to see what those past use-cases were and whether it was something we want to support but in general I'd stick with something like the suggested wording above which would allow you to fetch arbitrary non-standard tag files.